src/lambda/release.js

```diff
@@ -0,0 +1,76 @@
+import fetch from 'node-fetch';
+
+exports.handler = async event => {
+	const { version, url } = await fetchRelease(
+		`preactjs/${event.queryStringParameters.repo || 'preact'}`
+	);
+
+	return {
+		statusCode: 200,
+		body: JSON.stringify({
+			version,
+			url
+		}),
+		headers: {
+			'Cache-Control': 'public, s-maxage=1800',
+			'Access-Control-Allow-Origin': 'http://localhost:8080'
```

Comment:
This is to make stuff work in `localhost` a better alternative would be to use `netlify-dev` but that requires people to log into netlify which doesn't feel ideal for an OSS project

Reply:
Agree, that's a better compromise 👍